integration/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "integration-tests",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "test": "npm install && mocha test/*",
+    "test:fast": "mocha test/*"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
why ISC?

Reply:
Forgot to check these info, will update them.

Reply:
Done